### PR TITLE
Add tool tip to ViewModeSwitcher.

### PR DESCRIPTION
**Pull request by raviks789**
Tool tip to ViewModeSwitcher has been added here.

#144

### CODE REVIEW

File: library/Icingadb/Widget/ViewModeSwitcher.php [hunk 1]
```diff
@@ -154,6 +154,9 @@ protected function assemble()
 
             if ($viewMode === $currentViewMode) {
                 $input->getAttributes()->add('checked', true);
+                $label->getAttributes()->add('title', sprintf('%s view active', $viewMode));
```
Comment (flourish86) [inline library/Icingadb/Widget/ViewModeSwitcher.php:157]: Please start tool tip sentence with Uppercase letter.

**Review by flourish86 — CHANGES_REQUESTED** [review]

File: library/Icingadb/Widget/ViewModeSwitcher.php [hunk 2]
```diff
@@ -154,6 +154,9 @@ protected function assemble()
 
             if ($viewMode === $currentViewMode) {
                 $input->getAttributes()->add('checked', true);
+                $label->getAttributes()->add('title', t(sprintf('%s view active', ucfirst($viewMode))));
```
Comment (nilmerg) [inline library/Icingadb/Widget/ViewModeSwitcher.php:157]: Three issues regarding localisation:

a) gettext will parse the source code to extract messages, so translatable strings need to be directly wrapped by the translation function (i.e. `t('%s view active')`)
b) Passing a computed result to any translation function isn't possible. gettext extracts the arguments for such translation functions as `t()` and hence computed values won't be.. computed :smile: 
c) Translatable messages may of course contain placeholders for sprintf, but then it's usually a value that is localized differently (such as a date) or not localized at all (such as a number). This however is a name, and names are localizable, they're adjectives after all. So here we'd need three different messages, one for each mode.

File: library/Icingadb/Widget/ViewModeSwitcher.php [hunk 3]
```diff
@@ -154,6 +154,9 @@ protected function assemble()
...
+            } else {
+                $label->getAttributes()->add('title', t(sprintf('Switch to %s view', $viewMode)));
```
Comment (nilmerg) [inline library/Icingadb/Widget/ViewModeSwitcher.php:159]: See above.

**Review by nilmerg — CHANGES_REQUESTED** [review]

**Review by flourish86 — APPROVED** [review]

File: library/Icingadb/Widget/ViewModeSwitcher.php (hunk 2 above)
Comment (nilmerg) [reply library/Icingadb/Widget/ViewModeSwitcher.php]: This is resolved? You've only applied a) and b), what about c)?

**Review by nilmerg — APPROVED** [review]

**Pull request merged**